ports/libmad/vcpkg.json

```diff
@@ -13,5 +13,11 @@
       "name": "vcpkg-cmake-config",
       "host": true
     }
-  ]
+  ],
+  "features": {
```

Comment:
Ah, we need probably need something along the lines of

```suggestion
  "default-features": [
    {
      "name": "aso",
      "platform": "x86 | x64 | arm"
    }
  ],
  "features": {
```

Reply:
(which should be upstreamed too)

Reply:
Or just revert the introduction, because it can be always enabled on all platforms and there is not really a point for exposing this as feature.

Reply:
Hm, making it a feature felt cleaner to me, that way the build configuration is more transparent to vcpkg and we avoid hacking around with CMake options where it's not needed (we unfortunately do anyway with the FPM flags, but I don't see a good way around that without heavily patching the upstream CMakeLists).

Reply:
Since upstream already merged it, I would add the default features in a PR and then cherry-pick them here again. The feature might not be super useful, but having slightly more configurability than needed shouldn't hurt (I've always appreciated when ports already provide features for this kind of stuff, that made e.g. #133 a simple matter of updating the platform clauses).

Reply:
OK.

Reply:
- https://github.com/microsoft/vcpkg/pull/37154